test/examples/examples.jl

```diff
@@ -2,6 +2,121 @@
+                                                            "hydrostatic_water_column_2d.jl");
+                                                   kwargs...)
+
+                    @test sol.retcode == ReturnCode.Success
+                    @test count_rhs_allocations(sol, semi) == 0
+                end
+            end
+        end
+
+        @trixi_testset "fluid/hydrostatic_water_column_2d.jl (EDAC) " begin
+            # import variables into scope
+            trixi_include(@__MODULE__,
+                          joinpath(examples_dir(), "fluid",
+                                   "hydrostatic_water_column_2d.jl"),
+                          fluid_system=nothing, sol=nothing, semi=nothing, ode=nothing)
+
+            viscosity = ViscosityAdami(nu=alpha * smoothing_length * sound_speed / 8)
+            fluid_system = EntropicallyDampedSPHSystem(tank.fluid, smoothing_kernel,
+                                                       smoothing_length, sound_speed,
+                                                       viscosity=viscosity,
+                                                       density_calculator=ContinuityDensity(),
+                                                       acceleration=(0.0, -gravity))
+
+            hydrostatic_water_column_tests = Dict(
+                "default" => (),
+                "with source term damping" => (source_terms=SourceTermDamping(damping_coefficient=1e-4),),
+                "with SummationDensity" => (fluid_density_calculator=SummationDensity(),
+                                            clip_negative_pressure=true),
+                "with ViscosityAdami" => (
+                                          # from 0.02*10.0*1.2*0.05/8
+                                          viscosity=ViscosityAdami(nu=0.0015),),
+                "with ViscosityMorris" => (
+                                           # from 0.02*10.0*1.2*0.05/8
+                                           viscosity=ViscosityMorris(nu=0.0015),),
+                "with ViscosityAdami and SummationDensity" => (
+                                                               # from 0.02*10.0*1.2*0.05/8
+                                                               viscosity=ViscosityAdami(nu=0.0015),
+                                                               fluid_density_calculator=SummationDensity(),
+                                                               clip_negative_pressure=true),
+                "with ViscosityMorris and SummationDensity" => (
+                                                                # from 0.02*10.0*1.2*0.05/8
+                                                                viscosity=ViscosityMorris(nu=0.0015),
+                                                                fluid_density_calculator=SummationDensity(),
+                                                                clip_negative_pressure=true),
```

Comment:
EDAC doesn't need a state equation, so `clip_negaive_pressure` is not necessary

Also, you define the system above with viscosity and density calculator and pass it to `trixi_include`. The `kwargs` from the `Dict` are then never used in the example, right?

Reply:
How about one test set called `hydrostatic water column`. Above you can then create the dict for both edac and wcsph.